pkg/asset/agent/image/ignition.go

```diff
@@ -382,6 +382,7 @@ func getDefaultEnabledServices() []string {
 		"oci-eval-user-data.service",
 		"iscsistart.service",
 		"set-hostname.service",
+		"iscsiadm.service",
```

Comment:
is there a benefit to run `iscsiadm` instead of enabling `iscsid` (and perhaps configure `iscsistart` after `iscsid` has started)  directly?

Reply:
According to [RHEL 9 doumentation] it seems iscsiadm should be used to start iscsid (https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/managing_storage_devices/configuring-an-iscsi-initiator_managing-storage-devices#configuring-an-iscsi-initiator_managing-storage-devices)

`By default, an iSCSI service is lazily started and the service starts after running the iscsiadm command. If root is not on an iSCSI device or there are no nodes marked with node.startup = automatic then the iSCSI service will not start until an iscsiadm command is executed that requires iscsid or the iscsi kernel modules to be started.`

I've also confirmed it works by using `ExecStart=systemctl start iscsid.service` instead of iscsiadm in the service however.